storybookjs/storybook
# Add support for importing markdown in storybook

Issue:

## What I did
Added support for markdown (parse to html string) in react storybook at the webpack level

## How to test
Import a markdown file and assert that it has paresed as expected

Is this testable with jest or storyshots?
No

Does this need a new example in the kitchen sink apps?
Yes.

Does this need an update to the documentation?
Yes.

## Screenshot from public site
![screen shot 2017-11-15 at 9 15 42 am](https://user-images.githubusercontent.com/4509692/32841429-483e6d52-c9e8-11e7-8d43-863dbbccdf9a.png)
